apps/dashboard/config/api_docs/holidays.yml

```
@@ -88,6 +89,133 @@ endpoints:
+              { country: 'AR', year: 2024 }
+            ]
+          })
+        });
+
+        const { data } = await response.json();
+        data.results.forEach(r => {
+          if (r.found) console.log(`${r.country} ${r.year}: ${r.total} holidays`);
+          else console.log(`${r.country} ${r.year}: not found`);
+        });
+
     code_examples:
```

YAMLlint (1.38.0)

[error] duplication of key "code_examples" in mapping

(key-duplicates)